portal/app/core/assets/js/screen2.js

```diff
@@ -0,0 +1,784 @@
+var diagram = new SimpleDiagram('#diagram', {
```

Comment:
Also, please rename screen2 into something more meaningful

Reply:
Renamed it to controller-diagram, referring to a controller for the simple-diagram.js file